test/GMG/mpi/GMGLinearSolversPoissonTests.jl

```diff
@@ -72,6 +72,8 @@ module GMGLinearSolverTests
     cmodel=CartesianDiscreteModel(domain,coarse_grid_partition)
     mh=ModelHierarchy(parts,cmodel,num_parts_x_level)
 
+    if GridapP4est.i_am_in(mh.level_parts[1])
```

Comment:
Wouldnt it be better to handle this at the library level (if possible) instead of transferring the responsibility to the user driver?